cmd/trivy.go

```diff
@@ -0,0 +1,23 @@
+package cmd
+
+import (
+	"dynamic-buildkite-template/generator"
+)
+
+var (
+	trivyPluginConfig generator.TrivyPluginConfig
+	flagPrefix        = "trivy-"
+)
+
+func init() {
+	generateCmd.Flags().Int(flagPrefix+"exit-code", 0, "Controls whether the security scan is blocking or not for trivy buildkite plugin")
+	generateCmd.Flags().String(flagPrefix+"timeout", "", "Controls the maximum amount of time a scan will run for trivy buildkite plugin")
+	generateCmd.Flags().String(flagPrefix+"severity", "UNKNOWN,LOW,MEDIUM,HIGH,CRITICAL", "Controls the severity of the vulnerabilities to be scanned for trivy buildkite plugin")
```

Comment:
```suggestion
	generateCmd.Flags().String(flagPrefix+"severity", "HIGH,CRITICAL", "Controls the severity of the vulnerabilities to be scanned for trivy buildkite plugin")
```

Lets use the defaults we would expect set

Reply:
completed as per the suggestion,Tq!